jetty-http/src/test/java/org/eclipse/jetty/http/MultiPartFormInputStreamTest.java

```diff
@@ -69,8 +69,7 @@ public MultiPartFormInputStreamTest()
     }
     
     @Test
-    public void testBadMultiPartRequest()
-            throws Exception
```

Comment:
There are no actual changes to this file, other than formatting changes. This clutters up the PR and makes it hard to follow what changes were actually necessary for the bug fix, so can we leave formatting changes to be done only where there is a substantive change to the file.